boilerplate/config.json

```diff
@@ -79,7 +79,7 @@
         }
     },
     "metaDataTypes": {
-        "documentOnRetrieve": ["accountUser", "role", "dataExtension"],
```

Comment:
why remove that here? this is used only if the respective type is actually retrieve. not having accountUser here disables the automatic documentation when somebody does retrieve accountUser